<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Renamer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Новое название:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameBeforeIndex"/>
      </item>
      <item>
       <widget class="QSpinBox" name="indexSpinBox">
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameAfterIndex"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="movement">
       <enum>QListView::Static</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="renameButton">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Переименовать</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файлы</string>
    </property>
    <addaction name="openFilesAction"/>
    <addaction name="rotateListAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="openFilesAction">
   <property name="text">
    <string>Открыть файлы</string>
   </property>
  </action>
  <action name="rotateListAction">
   <property name="text">
    <string>Развернуть список файлов</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
